downstream/modules/automation-hub/proc-configure-automation-hub-server-gui.adoc

```diff
@@ -0,0 +1,51 @@
+[id="proc-configure-automation-hub-server-gui"]
+== Configuring Red Hat {HubName} as the primary source for content using the web console
+
+Configure Red Hat automation hub as your primary source of content by using the web console. To configure automation hub, you must create a credential and add it to the Organization’s Galaxy Credentials field. With automation hub, you have access to certified, supported collections.
+
+.Prerequisites
+
+* You have obtained the API token for the {HubName} server. See link: https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/1.2/html/getting_started_with_red_hat_ansible_automation_hub/proc-create-api-token[Creating the {HubName} API token] for more information.
+[IMPORTANT]
+====
+	Creating a new token revokes any previous tokens 
+	generated for Private Automation Hub. Ensure that you update any Controller or scripts that you created with the previous token.
+====
+
+.Procedure
+
+. Navigate to your Automation Controller. 
+. Create a new credential.
+.. Click btn:[Add] from the **Credentials** screen.
+.. Enter the name for your new credential in the **Name** field.
+.. Optional: enter a description and enter or select the name of the organization with which the credential is associated.
+.. Under Organization, select the organization that you would like to use the credential for.
+.. Select **Ansible Galaxy/Automation Hub API Token** as the credential type.
+.. Under **Type Details**, enter the **Galaxy Server URL**, **Authentication Server URL**, and **API Token** created in the Prerequisite.
+.. Click btn:[Save].
+. Select the credential you created from the Organization’s Galaxy Credentials field.
+.. Navigate to menu:Access[Organizations].
+.. Select the organization where you’d like to add your Galaxy credentials.
+.. Click btn:[Edit].
+.. Under Galaxy Credentials, click image:search-icon.png.
+.. Select the credential you created for automation hub, and place it at the beginning of the list.
+.. Optional: If you have a secondary source of content, such as Ansible Galaxy, place this credential after the credential you created for automation hub. 
+.. Click btn:[Select].
+.. Click btn:[Save].
+
+.Verification
+
+To validate the credential, update an existing source control management (SCM)-based project by selecting the project and clicking the [refresh icon] button.
```

Comment:
Is this `[refresh icon] button` -> **Refresh** icon?
Same comment for 'Update the project by clicking the refresh icon [refresh icon].' text.